aces/Loris
# [API] Changing Candidate Visit view to allow v0.0.2

## Brief summary of changes

Change to the Candidate Visit endpoint to allow the methods for version 0.0.2 of the API, which did not have projects.

#### Testing instructions

1. Go to <HOST>/api/v0.0.3/candidates/300001/V1

2. Go to <HOST>/api/v0.0.2/candidates/300001/V1. The page should display the same information, except `v0.0.2` should not contain the field `Project`

#### Link to the related issue

* Resolves #6058
